build.sbt

```diff
@@ -10,21 +10,27 @@ scalaVersion := "2.12.8"
 libraryDependencies += guice
 libraryDependencies += "org.scalatestplus.play" %% "scalatestplus-play" % "4.0.3" % Test
 
-// Adds additional packages into Twirl
-//TwirlKeys.templateImports += "tna.controllers._"
-
-// Adds additional packages into conf/routes
-// play.sbt.routes.RoutesKeys.routesImport += "tna.binders._"
-
 val playPac4jVersion = "8.0.1"
 val pac4jVersion = "3.7.0"
+val akkaVersion = "2.6.3"
 
 libraryDependencies ++= Seq(
   "org.pac4j" %% "play-pac4j" % playPac4jVersion,
   "org.pac4j" % "pac4j-http" % pac4jVersion,
   "org.pac4j" % "pac4j-oidc" % pac4jVersion exclude("commons-io", "commons-io"),
   "com.typesafe.play" % "play-cache_2.12" % "2.7.2",
-  "org.apache.shiro" % "shiro-core" % "1.4.0"
+  "org.apache.shiro" % "shiro-core" % "1.4.0",
+  "net.bytebuddy" % "byte-buddy" % "1.9.7",
+  "io.circe" %% "circe-core" % "0.13.0",
+  "io.circe" %% "circe-generic" % "0.13.0",
+  "com.softwaremill.sttp.client" %% "core" % "2.0.0-RC9",
+  "com.softwaremill.sttp.client" %% "circe" % "2.0.0-RC9",
+  "com.softwaremill.sttp.client" %% "async-http-client-backend-future" % "2.0.0-RC9",
+   "uk.gov.nationalarchives" %% "tdr-graphql-client" % "0.0.1",
```

Comment:
Extra indentation on this line. Out of curiosity, does the code linter pick this up?

Reply:
No, it doesn't seem to, I think it only checks in `src/main/*` I've corrected this anyway.